src/main/java/io/github/haykam821/withersweeper/game/phase/WithersweeperActivePhase.java

```diff
@@ -208,6 +212,31 @@ private ActionResult useBlock(ServerPlayerEntity uncoverer, Hand hand, BlockHitR
 		return result;
 	}
 
+	private void autoReveal(ServerPlayerEntity uncoverer, BlockPos clickedBlockPos, Field clickedField) {
+		if (clickedField.getBlockState() == Blocks.WHITE_WOOL.getDefaultState()) {
```

Comment:
Check if this is a number field with the value `0` rather than checking the world representation.